### PR TITLE
precipitable water import

### PR DESCRIPTION
**Pull request by PharaohCola13**
#### Description Of Changes
Updates to wyoming.py to allow Precipitable Water measurements to be pulled.

### CODE REVIEW

File: siphon/http_util.py [hunk 1]
```diff
@@ -18,11 +18,10 @@
 
 import requests
 
-from . import __version__
+# from . import __version__
```
Comment (dopplershift) [inline siphon/http_util.py]: Why are you commenting out this line?

File: siphon/simplewebservice/wyoming.py [hunk 2]
```diff
@@ -11,8 +11,12 @@
 import numpy as np
 import pandas as pd
 
-from .._tools import get_wind_components
-from ..http_util import HTTPEndPoint
+import sys, os
+sys.path.insert(0, os.path.abspath('..'))
```
Comment (dopplershift) [inline siphon/simplewebservice/wyoming.py]: I’m guessing this is working around problems trying to run siphon from git on your system? Did you properly install Siphon as described in the [Contributor’s Guide](https://github.com/Unidata/siphon/blob/master/CONTRIBUTING.md#pull-requests)?

File: siphon/simplewebservice/wyoming.py [hunk 3]
```diff
@@ -93,14 +97,15 @@ def _get_data(self, time, site_id):
         latitude = float(lines[4].split(':')[1].strip())
         longitude = float(lines[5].split(':')[1].strip())
         elevation = float(lines[6].split(':')[1].strip())
+        pw          = float(lines[-1].split(':')[1].strip())
```
Comment (dopplershift) [inline siphon/simplewebservice/wyoming.py]: Don’t worry about trying to line things up, just put a single space on either side of the equal sign.

**Comment by PharaohCola13** [comment]
The suggestions that were laid were some cheap workarounds for local testing/running. Those changes have been adjusted such that the only real alteration is the inclusion of precipitable water in the data pull, as that was the goal.

Thank you

**Comment by dopplershift** [comment]
If you want bonus points, you could test that the value of precipitable water is properly decoded here: 

https://github.com/Unidata/siphon/blob/f39e322dcf120497a295b23bafffddda2febc112/siphon/tests/test_wyoming.py#L30-L37

**Comment by PharaohCola13** [comment]
I can take a look and integrate precipitable water into the tests.

**Pull request merged**